### ISSUE TITLE
caffe to pytorch: torch cannot load converted model

### ISSUE THREAD

**Issue by Microos** [Microsoft/MMdnn#146]
Platform (like ubuntu 16.04/win10):
`Ubuntu 16.04`
Python version:
`3.5`
Source framework with version (like Tensorflow 1.4.1 with GPU):
`caffe`
Destination framework with version (like CNTK 2.3 with GPU):
`pytorch 0.4.0`
Pre-trained model path (webpath or webdisk path):
`None`

Running scripts:
`mmconvert --srcFramework caffe --inputWeight ./x0316_iter_15000.caffemodel --inputNetwork test.prototxt --dstFramework pytorch --outputModel x0316.pth`

Execute the above command will give me the `x0316.pth`  as expected.  The output:
```
IR network structure is saved as [d062f69da8de4ee49e02a402dd6dceb4.json].
IR network structure is saved as [d062f69da8de4ee49e02a402dd6dceb4.pb].
IR weights are saved as [d062f69da8de4ee49e02a402dd6dceb4.npy].
Parse file [d062f69da8de4ee49e02a402dd6dceb4.pb] with binary format successfully.
Target network code snippet is saved as [d062f69da8de4ee49e02a402dd6dceb4.py].
Target weights are saved as [d062f69da8de4ee49e02a402dd6dceb4.npy].
PyTorch model file is saved as [x0316.pth], generated by [d062f69da8de4ee49e02a402dd6dceb4.py] and [d062f69da8de4ee49e02a402dd6dceb4.npy].
```

When I try to load the model:
```Python
pth = './x0316.pth'
import torch as tc
model = tc.load(pth)
```
It raises error:
```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-3-d440d4d079da> in <module>()
----> 1 torch.load(pd)

~/.local/lib/python3.5/site-packages/torch/serialization.py in load(f, map_location, pickle_module)
    301         f = open(f, 'rb')
    302     try:
--> 303         return _load(f, map_location, pickle_module)
    304     finally:
    305         if new_fd:

~/.local/lib/python3.5/site-packages/torch/serialization.py in _load(f, map_location, pickle_module)
    467     unpickler = pickle_module.Unpickler(f)
    468     unpickler.persistent_load = persistent_load
--> 469     result = unpickler.load()
    470 
    471     deserialized_storage_keys = pickle_module.load(f)

ImportError: No module named 'd062f69da8de4ee49e02a402dd6dceb4'

```

**Comment by zhantong** [comment]
Hi @Microos , the issue has been fixed in #154 , have a try please.

Now the script will produce  two files `x0316.pth` and `x0316.py`. Keep these two files in the CWD, `tc.load('x0316.pth')` will no longer raise errors.

**Comment by Microos** [comment]
@zhantong Thank you for your update, I will have a try soonly.
